app/src/test/java/com/github/se/orator/model/chatGPT/ChatViewModelTest.kt

```diff
@@ -38,99 +38,118 @@ class ChatViewModelTest {
-                   - Incorporate realistic objections, questions, or concerns that align with the user's product (${salesContext.product}) and its key features (${salesContext.keyFeatures.joinToString(", ")}).
-                
-                2. **Session Structure**:
-                   - Start with an introduction, setting the stage for the negotiation (e.g., a corporate buyer looking to evaluate the product).
-                   - Allow the user to deliver their pitch, covering the product's key features and value proposition.
-                   - Gradually introduce objections or negotiation challenges to test the user's ability to adapt and persuade.
-                
-                3. **Focus Areas**:
-                   - Evaluate the clarity of the pitch and how well it addresses the client's needs.
-                   - Assess the user's handling of objections and ability to build rapport.
-                   - Observe how effectively the user uses persuasive techniques to close the deal.
-                
-                4. **Professional Role**:
-                   - Maintain a realistic persona throughout the session.
-                   - Do not provide feedback or hints during the pitch. Focus on creating an immersive experience.
-                
-                5. **Post-Session Feedback**:
-                   - Feedback will be provided separately at the end of the session.
-                
-                Start the session by introducing the negotiation scenario, your role, and setting expectations for the user.
-                """
-          .trimMargin()
+        You are simulating a sales negotiation practice session for the user, who is preparing to pitch the product/service "Marketing Services" to Potential Clients. The primary goal is to Close the deal. Your objective is to provide a realistic and challenging sales scenario. Follow these detailed guidelines:
+
+        1. **Scenario Setup**:
+           - Role-play as a potential client from the target audience (Potential Clients).
+           - Incorporate anticipated challenges: Budget Constraints, Competition.
+           - Focus on the negotiation aspect: Handling Objections.
+
+        2. **Session Structure**:
+           - Allow the user to deliver their pitch, emphasizing key features: Customized Strategies, ROI Focused.
+           - Introduce objections or negotiation hurdles relevant to the scenario.
+
+        3. **Focus Areas**:
+           - Evaluate the user's ability to handle objections and close the deal.
+           - Observe their use of persuasive language and negotiation techniques.
+
+        4. **Professional Role**:
+           - Maintain the persona of a realistic client.
+           - Do not provide feedback or hints during the session.
+
+        Start the session by setting the scene and initiating the conversation.
+    """
+          .trimIndent()
```

Comment:
The tests align perfectly with the changes made and ensure that the expected behavior is verified. Excellent job!

Reply:
Thanks!